README.md

```diff
@@ -58,7 +58,8 @@ Of course you can change `corp-release` to any name you like.
 * `-d` or `--dryrun`: it runs in non-destructive mode. No alteration should be done in your workspace.
 * `--pre-commit [npm-script]`: Pre-commit hook. Pass the name of the npm script to run. It will run like this: `npm run [npm-script]`.
 * `--post-success [command]`: Post-success hook (after `git push` completes successfully). Pass a command to run as the argument. Eg: `--post-success "npm publish"`.
-* `-b [branch]` or `--branch [branch]`: Branch name allowed to run release. Default is `master`. If you want to release from another branch, you need to specify. 
+* `-b [branch]` or `--branch [branch]`: Branch name allowed to run release. Default is `master`. If you want to release from another branch, you need to specify.
+* `-B` or `--any-branch`: Allows release from any branch. Useful for Jenkins as git does a revision check-out.
```

Comment:
Yes, it sounds like a good idea to me. What do you think @enriquecaballero ?
It keeps API the same and less options to document and and maintain.